hbase_sql/src/main/java/com/splicemachine/derby/stream/spark/SparkScanSetBuilder.java

```diff
@@ -102,6 +101,14 @@ else if (storedAs.equals("O"))
                 // The predicates have variant qualifiers (or we are reading ORC with our own reader), we couldn't push them down to the scan, process them here
                 return locatedRows.filter(new TableScanPredicateFunction<>(operationContext));
             }
+            if (dsp.isSparkExplain()) {
```

Comment:
You could return an empty dataset instead if this could be a problem, sampling might actually read the data from Splice anyway

Reply:
Thanks, I'll keep this in mind.  So far it's not an issue.  I'm not sure if this would change what's printed in the spark explain, for example, would it still indicate which parquet table is being read from for external tables, etc.

Reply:
That's a good point, it wouldn't.